h/static/scripts/directive/annotation.js

```diff
@@ -360,7 +372,7 @@ function AnnotationController(
       // Highlights are always private.
       domainModel.permissions = permissions.private();
       domainModel.$create().then(function() {
-        $rootScope.$emit('annotationCreated', domainModel);
+        $rootScope.$emit(events.ANNOTATION_CREATED, domainModel);
```

Comment:
Is it necessary to close an existing empty unsaved annotation on creating a new highlight? Creating a highlight won't result in two open annotation editors